src/python/WMCore/Alerts/EmailAlert.py

```diff
@@ -0,0 +1,40 @@
+"""
+EmailAlert - send alerts via email
+"""
+
+from __future__ import division
+import smtplib
+import logging
+
+
+class EmailAlert:
```

Comment:
Eric, given that it does not rely on any WMCore related code, how about we move it under src/python/Utils ? It would be easier for others to import from WMCore